eclipse-bluechi/README.md

```diff
@@ -0,0 +1,206 @@
+# Eclipse BlueChi
+
+![Smart trailer blueprint](../docs/diagrams/bluechi.png)
```

Comment:
Depending on in which order this PR and #19 get merged, this path may need to be updated

Reply:
@ladatz @odra do you have a preference of the order in which I should then merge #18 and #19? Considering the comment by @ladatz, I guess that this link needs to adapted anyway, once #19 gets merged.

Reply:
You can merge #19, I will rebase my pr.